pvmismatch/tests/test_pvmodule.py

```diff
@@ -1,38 +1,33 @@
 """
 Tests for pvmodules.
 """
-
-from nose.tools import ok_
+import pytest
 from pvmismatch.pvmismatch_lib.pvmodule import PVmodule, TCT492, PCT492
 from pvmismatch.pvmismatch_lib.pvcell import PVcell
 import numpy as np
 from copy import copy
 
-
 def test_calc_mod():
     pvmod = PVmodule()
-    ok_(isinstance(pvmod, PVmodule))
```

Comment:
FYI: if these were existing tests, then there's no reason to change them, unless you want to remove `nose` as a dependency` which there's no need to do. [pytest will run nose tests](https://docs.pytest.org/en/latest/nose.html?highlight=nose) as well as the [simple Python `assert` statement](https://docs.python.org/3/reference/simple_stmts.html?highlight=assert#the-assert-statement)

Reply:
@chetan201 PTAL